source/jormungandr/jormungandr/schedule.py

```diff
@@ -89,11 +89,58 @@ def update_passages(stop_schedule, next_realtime_passages):
         new_dt.realtime_level = type_pb2.REALTIME
 
 
+def _update_passages(passages, route_point, next_realtime_passages):
+    if next_realtime_passages is None:
+        return
+
+    # create the template for the realtime passages
+    template = deepcopy(passages[0])
+    template.pt_display_informations.ClearField("headsign")
+    template.pt_display_informations.ClearField("direction")
+    template.pt_display_informations.ClearField("physical_mode")
+    template.pt_display_informations.ClearField("description")
+    template.pt_display_informations.ClearField("uris")
+    template.pt_display_informations.ClearField("has_equipments")
+    del template.pt_display_informations.messages[:]
+    del template.pt_display_informations.impacts[:]
+    del template.pt_display_informations.notes[:]
+    del template.pt_display_informations.headsigns[:]
+    template.stop_date_time.ClearField("arrival_date_time")
+    template.stop_date_time.ClearField("departure_date_time")
+    template.stop_date_time.ClearField("base_arrival_date_time")
+    template.stop_date_time.ClearField("base_departure_date_time")
+    template.stop_date_time.ClearField("properties")
+
+    # filter passages with entries of the asked route_point
+    idx_to_del = []
+    for i, p in enumerate(passages):
```

Comment:
I think it would be good to have a filter_method that takes a collection and a lambda to do this